src/test/regress/expected/multi_connection_cache.out

```diff
@@ -104,3 +108,12 @@ SELECT get_and_purge_connection('localhost', :worker_port);
 (1 row)
 
 SET client_min_messages TO DEFAULT;
+\c
+-- purge existing connection to localhost
+SELECT connect_and_purge_connection('localhost', :worker_port);
+WARNING:  could not find hash entry for connection to "localhost:57638"
```

Comment:
Arg! I thought I took this out; missed it in the test output. I see now that I have such a check in both `Purge` methods. Fixed.